samples/choreo-token-exchange-react-app/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Add license.
Add a lint rule to use double quotes.
```suggestion
import React from "react";
```